backend/src/main/kotlin/com/hiper2d/auth/JwtConfigService.kt

```diff
@@ -0,0 +1,26 @@
+package com.hiper2d.auth
+
+import org.springframework.beans.factory.annotation.Value
+
+class JwtConfigService {
+
+    @Value("\${jwt.header}")
+    private var _tokenHeaderName: String = ""
```

Comment:
Why do you need both var and val?

Reply:
This looks ugly. But I don't know how else can I have **val** and **non nullable type** together with **@Value**

Reply:
@Value("\${jwt.header}")
    lateinit var tokenHeaderName: String

maybe somthing like that?

Reply:
But it's var, not final. The value can be changed